vscode-test-runner/workspace-1/.joyride/src/integration_test/nrepl_eval_test.cljs

```diff
@@ -0,0 +1,71 @@
+(ns integration-test.nrepl-eval-test
+  (:require [cljs.test :refer [testing is use-fixtures async]]
+            [integration-test.macros :refer [deftest-async]]
+            ["nrepl-client" :as nrepl-client]
+            [promesa.core :as p]
+            ["vscode" :as vscode]
+            ["util" :as util]))
+
+(def !state (atom {:client nil
+                   :client-eval nil}))
+
+(defn- from-nrepl-messages [messages k]
+  (.reduce messages
+           (fn [answer message]
+             (if (aget message k)
+               (str answer (aget message k))
+               answer))
+           ""))
+
+(use-fixtures :once
+  {:before
+   #(async done 
```

Comment:
I didn't know async/done is supported in fixtures. Where did you get that from? I think it's only supported in deftests (and only a single one per test).

Reply:
I got it from here: https://clojurescript.org/tools/testing (under **Async Fixtures**)

Reply:
Great, TIL!